runtime/j9vm/java11vmi.c

```diff
@@ -338,25 +338,14 @@ removePackageDefinition(J9VMThread * currentThread, J9Module * fromModule, const
 }
 
 static void
-trcModulesCreationPackage(J9VMThread * currentThread, J9Module * fromModule, const char *package)
+trcModulesCreationPackage(J9VMThread *currentThread, J9Module *fromModule, const char *package)
 {
 	PORT_ACCESS_FROM_VMC(currentThread);
-	J9InternalVMFunctions const * const vmFuncs = currentThread->javaVM->internalVMFunctions;
-	char moduleNameBuf[J9VM_PACKAGE_NAME_BUFFER_LENGTH];
-	char *moduleNameUTF = vmFuncs->copyStringToUTF8WithMemAlloc(
-		currentThread, fromModule->moduleName, J9_STR_NULL_TERMINATE_RESULT, "", 0, moduleNameBuf, J9VM_PACKAGE_NAME_BUFFER_LENGTH, NULL);
-
-	if (NULL != moduleNameUTF) {
-		if (0 == strcmp(moduleNameUTF, JAVA_BASE_MODULE)) {
-			Trc_MODULE_createPackage(currentThread, package, "java.base", fromModule);
-		} else {
-			Trc_MODULE_createPackage(currentThread, package, moduleNameUTF, fromModule);
-		}
-		if (moduleNameBuf != moduleNameUTF) {
-			j9mem_free_memory(moduleNameUTF);
-		}
-	} else {
-		vmFuncs->setNativeOutOfMemoryError(currentThread, 0, 0);
+	J9InternalVMFunctions const *const vmFuncs = currentThread->javaVM->internalVMFunctions;
```

Comment:
This variable is no longer used. Can it be removed? Also, applies to other `trcModules*` functions.

Reply:
Yes, a few of these may have been missed when I was extracting changes from the monolithic PR.